src/main/java/me/konsolas/conditionalcommands/placeholders/PlaceholderCooldown.java

```diff
@@ -0,0 +1,38 @@
+package me.konsolas.conditionalcommands.placeholders;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+import org.bukkit.entity.Player;
+
+import java.util.concurrent.TimeUnit;
+
+public class PlaceholderCooldown extends AbstractParameteredPlaceholder {
+    private final Cache<String, Long> cooldowns = CacheBuilder.newBuilder()
+            .expireAfterWrite(12, TimeUnit.HOURS)
+            .concurrencyLevel(1)
+            .build();
+
+    public PlaceholderCooldown() {
+        super("cooldown");
+    }
+
+    @Override
+    protected String getSub(Player player, String param) {
+        long currMillis = System.currentTimeMillis();
+        Long millis = cooldowns.getIfPresent(param);
+        if (millis == null) {
+            return "43200";
+        } else {
+            long diff = TimeUnit.MILLISECONDS.toSeconds(currMillis - millis);
+            if (diff < 1) {
+                return "0";
+            } else {
+                return Long.toString(diff);
```

Comment:
Could this return a value > 43200? It seems that Guava caches don't always perform evictions on read, so maybe it's worth putting a `Math.min(43200, diff)` here. See https://github.com/google/guava/wiki/CachesExplained#when-does-cleanup-happen